proto/orchestrator.proto

```diff
@@ -41,7 +41,7 @@ service Orchestrator {
       returns (AssessmentTool) {
     option (google.api.http) = {
       post : "/v1/orchestrator/assessment_tools"
-      body : "tool"
+      body : "*"
```

Comment:
I this case I think we said to keep it to `tool` for convience, so that REST users do not need the additional request wrappers, which is not intuitive